frontend/src/scenes/session-recordings/components/SimpleTimeLabel.tsx

```diff
@@ -0,0 +1,23 @@
+import { Dayjs, dayjs } from 'lib/dayjs'
+import { shortTimeZone } from 'lib/utils'
+
+function formatStringFor(d: Dayjs): string {
+    const today = dayjs()
+    if (d.isSame(today, 'year')) {
+        return 'MMM/DD, HH:mm:ss'
```

Comment:
Yeah, I always go yy mm dd hh mm ss cos i think it's the only logical choice in a world without agreement

but then i dropped the year because we're limited on space. I did have 11/12 but that's definitely not more clear 🤣

Reply:
No, you're right "Nov 26" could be November 2026 but "26 Nov" has only one meaning IMO